### PR TITLE
chore: use a license during e2e tests

### PR DESCRIPTION
**Pull request by ricardomaraschini**
in order to capture problems with the embedded license we need to use a version of the embedded cluster binary with an embedded kots license.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,4 +3,5 @@ bundle
 pkg/goods/bins
 pkg/goods/images
 *tgz
-license.yaml
```
Comment (laverya) [inline .gitignore:6]: why get rid of the `license.yaml` gitignore in the root?

Comment (laverya) [reply .gitignore:6]: ...because the make command no longer relies on it, got it

File: e2e/scripts/interactive-multi-node-install.exp [hunk 2]
```diff
@@ -21,4 +21,8 @@ expect -re "Add another node?.*:" { send "n\r" }
 set timeout 600
 expect "Enter a new Admin Console password:" { send "password\r" }
 expect "Confirm password:" { send "password\r" }
-expect "Admin Console is ready!"
+expect "Confirm password:" { send "password\r" }
```
Comment (laverya) [inline e2e/scripts/interactive-multi-node-install.exp:24]: should we really have to type the password three times?

Comment (ricardomaraschini) [reply e2e/scripts/interactive-multi-node-install.exp]: My mistake. Removed.

**Review by laverya — APPROVED** [review]

**Pull request merged**